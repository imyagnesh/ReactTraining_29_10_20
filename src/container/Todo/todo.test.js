import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Todo } from './index';

afterAll(cleanup);

describe('test todo component', () => {
  test('should take snapshot', () => {
    const { container } = render(<Todo />);
    expect(container).toMatchSnapshot();
  });

  test('should h1 render', () => {
    const { getByTestId } = render(<Todo />);
    const data = getByTestId('h1Test').innerHTML;
    expect(data).toStrictEqual('Todo Application');
  });
});
