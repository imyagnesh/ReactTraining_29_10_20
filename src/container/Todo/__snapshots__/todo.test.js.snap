// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test todo component should take snapshot 1`] = `
<div>
  <h1
    data-testid="h1Test"
  >
     Loading...
  </h1>
</div>
`;
